lightly_studio/.env.example

```
@@ -32,3 +32,16 @@ EXAMPLES_PASCALVOC_IMAGES_PATH="dataset_examples/voc2012_10_images/JPEGImages"
 EXAMPLES_PASCALVOC_MASKS_PATH="dataset_examples/voc2012_10_images/SegmentationClass"
 EXAMPLES_PASCALVOC_CATEGORIES_JSON_PATH="dataset_examples/voc2012_10_images/class_id_to_name.json"
 EXAMPLES_MIDV_PATH="dataset_examples/midv_2020_10_samples"
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)